pkg/template/worker_template.go

```diff
@@ -329,28 +314,13 @@ storage:
         name: giantswarm
       group:
         name: giantswarm
-  {{ if .ForceCGroupsV1 }}
-  filesystems:
-    - name: "OEM"
-      mount:
-        device: "/dev/disk/by-label/OEM"
-        format: "btrfs"
-  {{ end }}
   files:
     - path: /boot/coreos/first_boot
       filesystem: root
     {{ if .ForceCGroupsV1 }}
-    - filesystem: "OEM"
-      path: "/grub.cfg"
-      mode: 0644
-      append: true
-      contents:
-        source: "data:text/plain;base64,{{ index .Files "conf/grub.cfg" }}"
```

Comment:
Just a note from the off: I recommend to keep this because it helps to avoid the bind mount over /proc/cmdline on subsequent boots.

Reply:
Thanks for your comment @pothos 
Can you please explain how having the `grub.cfg` file in place avoids one bind mount? I'm not following

Reply:
The flag file under `/etc/` sets up a bind mount over `/proc/cmdline` to avoid the reboot. When the right parameters are set in `grub.cfg`, even this bind mount is not needed anymore.
In our testing the bind mount worked well, but it certainly is no long-term solution.

Reply:
ok thank you